base/common.py

```diff
@@ -28,13 +28,13 @@ def getSelectedObjs(select_tokens, env):
     select_tokens.append(LIT_STAR)
 
   current_view = env
-  for token in select_tokens:
+  for token in select_tokens[0]:
```

Comment:
Is this also a change brought by the update to pyparsing?

Reply:
> Is this also a change brought by the update to pyparsing?

Yes, I believe so.